kitchen.dokken.yml

```diff
@@ -11,6 +11,12 @@ provisioner:
   name: dokken
 
 platforms:
+  - name: amazonlinux-2
+    driver:
+      image: dokken/amazonlinux-2
+      pid_one_command: /usr/lib/systemd/systemd
+      intermediate_instructions:
+        - RUN touch ~/.bashrc
```

Comment:
These lines shouldn't be needed. Can you please remove them?

Reply:
without this line, the CI fails with `Error executing CLI: Did not find any shells to install`.
Look at this thread https://discuss.hashicorp.com/t/error-executing-cli-did-not-find-any-shells-to-install/15220/2. 
Looks like the `autocomplete` scans for bashrc and fails.

Reply:
Is this cookbook running the auto complete install? As I think that should only run with the repo install method. 

Either way it'd be better to add a check to the auto complete exec resource rather than adding it in to the CI environment as a real install will most likely have a bashrc present and it makes the cookbook more robust if it doesn't.

Reply:
@bmhughes - Removed from kitchen and added in the resource.

Reply:
I wouldn't add the bashrc, rather don't run the autocomplete install if it doesn't exist, otherwise you're adding unexpected functionality to the cookbook doing it this way. An only_if should be fine for this.  

Just for some context, the 'new direction' is for these cookbooks to purely be resource libraries, I wouldn't expect adding vault resources to suddenly change my bash configuration. Now personally I'll probably check the source first before using it, but history of complaints tells us a lot of people don't.